src/_includes/mac-path.md

```diff
@@ -0,0 +1,5 @@
+{{site.alert.important}}
+  Ensure that the **Homebrew `bin` directory is in your `PATH`** if you'd like
+  direct command-line access to Dart commands such as the Dart VM (`dart`). For
+  details, consult the [Homebrew FAQ](https://docs.brew.sh/FAQ).
+{{site.alert.end}}
```

Comment:
Note: While one my be tempted to instruct the read to add the Dart SDK `bin` folder to their path, that's not the usual way that things are done under Homebrew. Instead on configures `PATH` once to refer to the Homebrew `bin` directory. Following that, all of the standard packages installed with Homebrew will have their commands accessible.

Reply:
Where else is this file included? (If there's only one include, let's not make it a separate file, for simplicity's sake.)

Reply:
Oh, I thought that I had at least two files. I'll double check and merge the file if there is only one point of use.

Reply:
I think that I had originally anticipated including this file directly in `tutorials/server/get-started` (as well as `get-dart`). But having it only in `get-dart`, like we have it now in this PR, seems sufficient. Merging the file.